Flutter/lib/Animation.dart

```diff
@@ -22,14 +22,17 @@ class AutomatedAnimator extends StatefulWidget {
   @override
   _AutomatedAnimatorState createState() => _AutomatedAnimatorState();
 }
-class _AutomatedAnimatorState extends State<AutomatedAnimator> with SingleTickerProviderStateMixin {
+
+class _AutomatedAnimatorState extends State<AutomatedAnimator>
+    with SingleTickerProviderStateMixin {
```

Comment:
Why not keep this on one line? Seems appropriate to have it in one line

Reply:
Resolved in commit `app-style a06aa41`